hack/ci/cloud-init/controller.yaml.tpl

```diff
@@ -35,9 +35,12 @@
     ENABLED_SERVICES+=,g-api
 
     # Neutron
-    ENABLED_SERVICES+=,neutron-api,neutron-agent,neutron-dhcp,neutron-l3,neutron-trunk
+    ENABLED_SERVICES+=,q-svc,q-trunk,ovn-controller,ovs-vswitchd,ovn-northd,ovsdb-server,q-ovn-metadata-agent
+    Q_AGENT=ovn
```

Comment:
ovn is already the [default agent on zed](https://github.com/openstack/devstack/blob/stable/zed/lib/neutron_plugins/ml2#L11), we likely don't need to directly set this agent. Same for the `Q_ML2_PLUGIN_MECHANISM_DRIVERS` and `Q_ML2_TENANT_NETWORK_TYPE`